pagetags/views.py

```diff
@@ -19,4 +18,4 @@ def list_tags(request):
                 .filter(name__istartswith=q)
                 .values_list('name', flat=True))
     response = [ {'id':tag, 'label':tag, 'value':tag}  for tag in tags ]
-    return HttpResponse(json.dumps(response), mimetype='application/json')
+    return JsonResponse(response)
```

Comment:
Since `response` is a list, you need to add in the kwarg `safe=False`: https://docs.djangoproject.com/en/1.8/ref/request-response/#jsonresponse-objects